ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -73,8 +73,13 @@ private[testtool] abstract class CommandDeduplicationBase(
           // Submit command (first deduplication window)
           // Note: the second submit() in this block is deduplicated and thus rejected by the ledger API server,
           // only one submission is therefore sent to the ledger.
-          (_, completion1) <- submitRequestAndAssertCompletionAccepted(ledger)(request, party)
-          _ <- submitRequestAndAssertDeduplication(ledger)(request, party)
+          (offset1, completion1) <- submitRequestAndAssertCompletionAccepted(ledger, request, party)
```

Comment:
Nit: I think separate argument lists are slightly more readable.

Reply:
It would make sense for partial application but we don't really use it for `ledger` and I think having a separate list suggests otherwise.